online-fishing-store/src/app/pages/shopping-cart-page/shopping-cart-item/shopping-cart-item.component.html

```diff
@@ -0,0 +1,35 @@
+<div class="shopping-cart">
+  <div class="shopping-cart-product">
+    <img
+      class="shopping-cart-product__img"
+      [src]="shoppingCartProduct.img"
+      [alt]="productAltImgAttribute"
+    />
+    <div class="shopping-cart-product__info">
+      <span class="shopping-cart-product__name">{{ shoppingCartProduct.name }}</span>
+      <p class="shopping-cart-product__description">{{ shoppingCartProduct.description }}</p>
+      <div class="product__amount-price">
+        <label> Колличество:
+          <input 
+            class="input__amount" 
+            type="number"
+            min="1"
+            (keydown)="$event.preventDefault();"
+            [(ngModel)]="quantity"
+            name="amount"
+            (input)="addItemProduct()"
```

Comment:
You don't have to use here input event together with [(ngModel)]

In Angular if you use ngModel and you need event you should use "ngModelChange" event